raspberry/checkout/src/checkout_interactions.py

```diff
@@ -78,6 +92,18 @@ def set_pixels_color(self, color):
         self.pixels.fill(color)
         self.pixels.show()
 
+    def setup_encoder(self):
+        GPIO.add_event_detect(encoderLeft, GPIO.BOTH, callback=self.handle_encoder, bouncetime=50)
+        GPIO.add_event_detect(encoderRight, GPIO.BOTH, callback=self.handle_encoder, bouncetime=50)
+
+    def handle_encoder(self, channel):
+        if channel == encoderLeft:
+            print("Encoder Left")
+            self.on_prev()
```

Comment:
```suggestion
            if self.on_prev:
                self.on_prev()
```
don't wanna accidentally call it if for reasons unknown it wasn't assigned